src/tests/playwright/pages/util/audit.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Check if |actual| is close to |expected| using the given relative error
+ * |threshold|.
+ * @param {number} actual
+ * @param {number} expected
+ * @param {number} threshold
+ * @return {[boolean, string]} if |actual| is within |threshold| of |expected|
+ */
+export function beCloseTo(actual, expected, threshold) {
+  // The threshold is relative except when |expected| is zero, in which case
+  // it is absolute.
+  const absExpected = expected ? Math.abs(expected) : 1;
+  const error = Math.abs(actual - expected) / absExpected;
+  return [
+    error <= threshold,
+    // eslint-disable-next-line max-len
+    `${ actual } vs ${ expected } | error: ${error} | ${ Math.abs(actual - expected) } diff`,
+  ];
+}
+
+/**
+ * Assigns a given test function to the global window._webAudioTest property.
+ * This function is used to set up a test that can later be executed globally.
+ *
+ * @param {Promise<any>} testPromise - The test function to be assigned to
+ * window._webAudioTest.
+ */
+export const test = (testPromise) => {
+  window._webAudioTest = testPromise;
+  window._webAudioSuite && window._webAudioMutex();
+};
```

Comment:
Then I recommend using `isTestSuiteMode` instead of `webAudioSuite`. It emphasizes that the variable is a boolean flag.

Also your comment here can be added to the code. This explains why we need several variables clearly.